020-Getting-started/030-Framework-starter-guides/001-intro.mdx

```diff
@@ -18,3 +18,47 @@ The guides demonstrate the following features:
 - Generating TypeScript code with database branching
 - Querying and filtering
 - Using full-text fuzzy search
+
+## What you need to start?
+
+Each getting started guides requires you to install the Xata CLI, authenticate it, and store the CLI API key locally.
+
+### Xata CLI
+
+The [Xata command line interface (CLI)](/docs/getting-started/installation) allows you to work with your Xata serverless database directly from your command line and is important for integrating Xata's capabilities into your project. Using the CLI, you can manage your database, schemas, and data.
```

Comment:
```suggestion
The [Xata command line interface (CLI)](/docs/getting-started/installation) allows you to work with your Xata serverless database directly from your command line and is important for integrating Xata's capabilities into your project. Using the CLI, you can manage your database, schemas, and import data.
```

Since the CLI doesn't really provide capability to edit data, just to insert (from csv) it'd be best to not imply/build expectations for that here